packages/cli/test/prompts/prompt.test.js

```diff
@@ -162,16 +162,16 @@ describe('prompt', function() {
           });
           contracts.keyName.choices.should.deep.include.members([
             { name: 'Foo', value: 'Foo' },
-            { name: 'BarAlias[Bar]', value: 'BarAlias' },
+            { name: 'Bar', value: 'Bar' },
           ]);
         });
       });
 
       context('when looking for not yet added but built contracts', function() {
         it('returns an object with not added contracts', function() {
           const contracts = contractsList('keyName', 'Im a message', 'listy', 'notAdded');
-          contracts.keyName.choices.should.include.members(['Bar', 'Buz']);
-          contracts.keyName.choices.should.not.include('Foo');
+          contracts.keyName.choices.should.include.members(['Buz']);
+          contracts.keyName.choices.should.not.include.members(['Foo', 'Bar']);
```

Comment:
Do we know why `Bar` changed in this test?

Reply:
Because old implementation used aliases ` sinon.stub(ProjectFile.prototype, 'contracts').get(() => ({ Foo: 'Foo', BarAlias: 'Bar' }));` so `Bar` wouldn't be found because its alias is  `BarAlias`.